packages/zcli-themes/package.json

```diff
@@ -17,13 +17,15 @@
     "type:check": "tsc"
   },
   "dependencies": {
+    "@types/inquirer": "^8.0.0",
     "@types/ws": "^8.5.4",
     "axios": "^0.27.2",
     "chalk": "^4.1.2",
     "chokidar": "^3.5.3",
     "cors": "^2.8.5",
     "express": "^4.17.1",
     "glob": "^9.3.2",
+    "inquirer": "^8.0.0",
```

Comment:
Interesting addition! I had a look at [the Inquirer github project page](https://github.com/SBoudrias/Inquirer.js/), seems like a useful package.